docker/image/scc-metastore.Dockerfile

```diff
@@ -24,7 +24,7 @@ ENV HIVE_HOME=/opt/hive
 ENV HIVE_CONF_DIR=/etc/hive/conf
 
 RUN set -x && \
-    wget -q ${APACHE_MIRROR}/hive/hive-${HIVE_VERSION}/apache-hive-${HIVE_VERSION}-bin.tar.gz && \
+    wget -q https://archive.apache.org/dist/hive/hive-${HIVE_VERSION}/apache-hive-${HIVE_VERSION}-bin.tar.gz && \
```

Comment:
we have to get the hive 2.3.9 from apache archive site as the hive community removed such a version from the dln site.